### ISSUE TITLE
Targeting .NET Framework for a WPF application using Microsoft.Net.Sdk does not work

### ISSUE THREAD

**Issue by marcpopMSFT** [dotnet/sdk#13939]
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net472</TargetFramework>
    <UseWPF>true</UseWPF>
  </PropertyGroup>

</Project>
```
I get a bunch of compile errors around missing references. 
_The type or namespace name 'Data' does not exist in the namespace 'System.Windows' (are you missing an assembly reference?)	WpfApp1	C:\Users\marcpop\source\repos\WpfApp1\WpfApp1\MainWindow.xaml.cs_

Filing this so we can review and decide if this is a scenario we'd want to support. It only really matters for a customer multi-targeting between net5.0-windows and .NET framework.

**Comment by ryalanms** [comment]
This is an SDK issue tracked here: https://github.com/dotnet/sdk/issues/13939. 

To workaround the issue temporarily, use Microsoft.NET.Sdk.WindowsDesktop.  Thanks.

**Comment by sqeezy** [comment]
This just came up in on of my bigger projects. The biggest problem is the missing documentation on this. When converting an old style project to sdk style you can get all sorts of what seems like random fails. I was able to convert my project to sdk style and compile them in my IDE, but when I used the command line the build failed with weird compile errors.
In my opinion this hinders a smoother transition to dotnet 5. My plan was to first change the project style and then the platform. This is not possible due to this bug.
It would be very helpful to know whether this is going to be fixed.

**Comment by marcpopMSFT** [comment]
This got fixed in 5.0.200: https://github.com/dotnet/sdk/pull/15213
